docker/Dockerfile

```diff
@@ -25,6 +25,6 @@ COPY rust/src /src
 RUN touch /src/lib.rs
 RUN cargo build --all-features
 
-FROM ubuntu:latest
+FROM ubuntu:20.04
```

Comment:
Explicitly changed to latest LTS (`20.04`) as per [Dockerfile Lint](https://github.com/replicatedhq/dockerfilelint)'s recommendation! :-)